OpenRA.Mods.Common/Scripting/Properties/TransformProperties.cs

```diff
@@ -26,11 +27,19 @@ public TransformProperties(ScriptContext context, Actor self)
 			transforms = self.Trait<Transforms>();
 		}
 
+		[Obsolete("This function will be removed in future versions. Use Transform instead.")]
 		[ScriptActorPropertyActivity]
 		[Desc("Queue a new transformation.")]
 		public void Deploy()
 		{
 			transforms.DeployTransform(true);
```

Comment:
Can we add `TextNotificationsManager.Debug("Deploy is deprecated. Use Transform instead.");` here, similar to the now removed methods in the past? (E.g. https://github.com/OpenRA/OpenRA/commit/4ec19b34863532ecca1f123f327b43af441bda12)

Reply:
I don't think its necessary for this PR

Reply:
I'm not requesting this for other methods, but it should be done for this method, and in this PR as this is the PR which deprecates it. 😅

Reply:
Added. It is necessary for people only know about Lua to notice this change.